x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/create_trusted_app_form.tsx

```diff
@@ -129,6 +134,13 @@ const validateFormValues = (values: MaybeImmutable<NewTrustedApp>): ValidationRe
     );
   } else {
     values.entries.forEach((entry, index) => {
+      const isPathValid =
+        entry.type === 'wildcard'
```

Comment:
Correct me if I'm wrong, but I think you should check that the entry is a path right?
Something like `entry.field === ConditionEntryField.PATH ? ....`

Reply:
Yes. I check that where I push these soft warnings. I guess I can add the check here

Reply:
done be3415c43e62e555b9401f1b36e5eb50e7fa797f